synapse/third_party_rules/access_rules.py

```diff
@@ -33,6 +33,17 @@
     ACCESS_RULE_UNRESTRICTED,
 )
 
+# Rules to which we need to apply the power levels restrictions.
```

Comment:
```suggestion
# Rules to which we need to apply the power-levels restrictions.
#
```

Reply:
Spec says "power level" ftr

Reply:
fair enough. "power levels restrictions" is a bit ambiguous: "(power levels) restrictions" or "power (levels restrictions)" ? Hyphenating here seemed a good way to clarify.